app/gui/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -200,6 +200,8 @@ ensogl::define_endpoints! {
 
         set_view_mode        (view::Mode),
         set_profiling_status (profiling::Status),
+        /// Emit `on_port_press` for unit tests purposes.
+        test_port_press      (),
```

Comment:
I'd not write here "for unit test purposes". having an input `port_press` is a very valid thing and I like to ave it as part of the API. It should also not have the `test` prefix imo.